content/reference/deps_and_cli.adoc

```diff
@@ -336,6 +336,24 @@ Specify your AWS S3 credentials using one of these mechanisms:
 
 For more information, most of the advice in http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html[this AWS document] describes how credentials are located. Note however that the Java system properties options will NOT work with the command line tools (but would work if using the tools.deps.alpha library directly).
 
+==== Maven proxies
+
+In environments were the internet is accessed via a proxy, again existing Maven configuration in `~/.m2/settings.xml` is used to set up the proxy connection:
```

Comment:
were -> where

> again existing Maven configuration in `~/.m2/settings.xml` is used to set up the proxy connection:

This part of the sentence doesn't make sense to me. Especially the "again" part.

Reply:
Thank you, @bbatsov. I have fixed the typo and removed the word ‘again’. Suggestions welcome, the wording is taken from the similar sentence in section ‘Maven authenticated repos’.